### PR TITLE
Activate NetP waitlist for macOS

### PR DESCRIPTION
**Pull request by samsymons**
**Asana Task/Github Issue:** https://app.asana.com/0/0/1205400926163926/f

## Description

This PR activates the NetP waitlist for macOS at 1% rollout.

It targets macOS version 1.57.1 which was released Monday September 25, as this build includes necessary NetP stability improvements.

#### Reference
- [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
- [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
- [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

**Comment by samsymons** [comment]
⚠️ Let's not merge this until the Android team has signed off on it

**Review by SlayterDev — APPROVED** [review]

**Pull request merged**
